ExpenseTracker.Core.Tests.Unit/Services/Foundations/Transactions/TransactionServiceTests.Exceptions.Add.cs

```diff
@@ -26,10 +26,16 @@ public async Task ShouldThrowCriticalDependencyExceptionOnAddIfSqlErrorOccurrsAn
             SqlException sqlException = GetSqlException();
 
             var failedTransactionStorageException =
-                new FailedTransactionStorageException(sqlException);
+                new FailedTransactionStorageException(
+                    message: "Failed transaction storage error occurred, contact support.",
+                    innerException: sqlException
+                    );
```

Comment:
Move these all up to the end of previouc line.
There are many of these do it for all of them.

Reply:
@pratikkaje this occurs on many of the tests, maybe a copy paste operation result.
Fix these and I will take another review of the PR.

Reply:
@glhays I'm sorry, I'm not quite following the comment. Could you please clarify?

Reply:
Just move the ); on line 32 up to the end of 31. And all the others.

Reply:
Fixed. 👍